lib/app/features/home/match_details/match_details_screen.dart

```diff
@@ -17,11 +17,11 @@ class MatchDetailsScreen extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    final int teamID = liveMatch.fixture?.id ?? AppConstVariables.intPlaceholder;
+    final int matchID = liveMatch.fixture?.id ?? AppConstVariables.intPlaceholder;
```

Comment:
Same here - looks like there is no need to write int, you can use just final.

Reply:
done